backend/tests/restrictionTests.test.js

```diff
@@ -0,0 +1,80 @@
+//import axios from 'axios';
+var axios = require('axios')
```

Comment:
Codacy has a fix for the issue: [Replace `'axios')` with `"axios");`](https://app.codacy.com/gh/Harvir14/RecipEZ/pullRequest?prid=9801174)
```suggestion
var axios = require("axios");
```